### PR TITLE
Changed the convention for naming EGP in the manual author list.

### PR DESCRIPTION
**Pull request by egpuckett**
I prefer "Elbridge Gerry Puckett" to "E. Gerry Puckett".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,3 +51,8 @@
 /tests/*.x.prm
 /tests/lib*.so
 /Testing
+
+# Added by EGP for Clion
```
Comment (tjhei) [inline .gitignore]: can you change this to say ``# used for clion:``

Comment (egpuckett) [inline .gitignore]: Done!

**Comment by bangerth** [comment]
For future pull requests: You have two entirely independent issues that you address with this pull request -- the clion files and the naming issue. Please just open two separate pull requests for this. This makes it easier to review, and easier to attribute code pieces to pull requests (and their associated discussion) at a later time.

**Comment by egpuckett** [comment]
Got it! I thought about this when I pushed these changes to my origin. Is the best way to do this to make one commit on my local repository (i.e.,  laptop), thereby leaving some changed files in my local repository, and push that one commit to my origin, make the PR, and then proceed with the second changed file?

**Comment by bangerth** [comment]
Correct.

**Pull request merged**